dali/pipeline/operator/common.h

```diff
@@ -197,19 +196,18 @@ int GetShapeLikeArgument(std::vector<ExtentType> &out_shape,
         out_shape[i * ndim + d] = to_extent(tsvec[d]);
   }
 
-  return ndim;
+  return {batch_size, ndim};
 }
 
 template <typename ArgumentType = int, int out_ndim>
-int GetShapeArgument(TensorListShape<out_ndim> &out_tls,
-                      const OpSpec &spec,
-                      const std::string &argument_name,
-                      const ArgumentWorkspace &ws,
-                      int ndim = out_ndim,
-                      int batch_size = -1 /* -1 = get from "batch_size" arg */) {
-  ndim = GetShapeLikeArgument<ArgumentType>(out_tls.shape, spec, argument_name, ndim, batch_size);
-  out_tls.resize(out_tls.shape().num_samples / ndim, ndim);
-  return ndim;
+std::pair<int, int> GetShapeArgument(TensorListShape<out_ndim> &out_tls, const OpSpec &spec,
+                                     const std::string &argument_name, const ArgumentWorkspace &ws,
```

Comment:
Why the formatting change? This grouping seems random.

Reply:
CTRL+F did that. I can restore